Unified/checkor.py

```diff
@@ -1326,6 +1466,7 @@ def upward( ns ):
     parser.add_option('--tape_size_limit', help='The limit in size of all outputs',default=0,type=int)
     parser.add_option('--html',help='make the monitor page',action='store_true', default=False)
     parser.add_option('--no_report',help='Prevent from making the error report',action='store_true', default=False)
+    parser.add_option('--threads',help='Number of threads for processing workflows',default=1, type=int)
```

Comment:
Is it ok if we set the default to be 5?

Reply:
I have not done any scale or scaling test of this, so any number would do, once robustly tested

Reply:
@SvenjaPflitsch can you please test this on 1 workflow to see if it runs OK with multi threads?

Reply:
you obviously have to try with more than 1 workflow otherwise there is nothing to put on multiple threads ...